Documentation/source/Changelog.md

```diff
@@ -1,7 +1,14 @@
 Changelog
 =========
 
-4.7.0 (2019-04-17)
+4.7.2 (2019-05-03)
+------------------
+
+-   Renamed EPS QR Code url extraction name to `epsPaymentQRCodeUrl` , which is returned in the  `GiniVisionResultsDelegate.giniVisionAnalysisDidFinishWith(result:sendFeedbackBlock:)` 
```

Comment:
There's an extra whitespace become the comma.
Also, it's probably too details for a changelog.